dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -0,0 +1,165 @@
+[[nexus-maven-tutorial]]
+= Setting Up a Nexus Mirror for Maven
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[nexus-introduction]]
+== Introduction
+
+While developing your application with Java and Maven, you'll most likely be
+building many times. In order to shorten the build times of your pods, Maven
+dependencies can be cached in a local Nexus repository. This tutorial will step
+you through creating a Nexus repository on your cluster.
+
+This tutorial assumes that you'll be working with a project that is already set
+up for use with Maven. If you're interested in using Maven with your Java
+project, we highly recommend looking at
+link:https://maven.apache.org/guides/getting-started/index.html[their guide].
+
+In addition, be sure to check your application's image for Maven mirror
+capabilities. Many images that use Maven have a `MAVEN_MIRROR_URL` environment
+variable that you can use to simplify this process. If it doesn't have this
+capability, read
+link:https://books.sonatype.com/nexus-book/reference/config.html[the Nexus documentation]
+to configure your build properly.
+
+[[nexus-setting-up-nexus]]
+== Setting up Nexus
+
+Use `new-app` to download and deploy the official Nexus Docker image.
+
+----
+oc new-app sonatype/nexus
+----
+
+After the build has finished, run:
+
+----
+oc get svc
+----
+
+The output should resemble:
+
+----
+NAME      CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+nexus     172.30.170.249   <none>        8081/TCP   6s
+----
+
+Confirm that Nexus is running by navigating your browser to the new Cluster-IP
+at port 8081. The server may take a bit of time to start, be patient!
+
+Nexus comes pre-configured for the Central Repository, but you may need others
+for your application.  For many Red Hat images it is recommended to add the
+`jboss-ga` repository at `https://maven.repository.redhat.com/ga/`.
+Documentation on how to add resositories can be found
+link:https://books.sonatype.com/nexus-book/reference/config-maven.html[here].
+The default admin username is `admin`, and the password is `admin123`.
+
+[[nexus-adding-probes-to-check-for-success]]
+=== Using Probes to Check for Success
+
+This is a good time to set up readiness and livenes probes. These will
+periodically check to see that Nexus is running properly. More information on
+probes can be found xref:../application_health.adoc#dev-guide-application-health[here].
```

Comment:
Consider rephrasing to:

    xref:../application_health.adoc#dev-guide-application-health[Learn more about probes].